extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/MongoClientProcessor.java

```diff
@@ -66,6 +70,15 @@
     private static final DotName MONGO_CLIENT = DotName.createSimple(MongoClient.class.getName());
     private static final DotName REACTIVE_MONGO_CLIENT = DotName.createSimple(ReactiveMongoClient.class.getName());
 
+    static class MongoClientTracingEnabled implements BooleanSupplier {
+        MongoClientBuildTimeConfig mConfig;
+
+        @Override
+        public boolean getAsBoolean() {
+            return mConfig.tracingEnabled;
+        }
+    }
+
```

Comment:
I don't see this being used anywhere ?

Reply:
No, it is not. Remove it in the new push. Left the class in case we wanted the build step. But it makes no sense to keep it. Thanks